habit/src/components/screens/login/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { View, Text } from 'react-native'
+import CustomTextInput from '../../TextInput'
+
+export default function Login(props){
+    let input
+    return(
+        <View>
+            <Text>Hello login form</Text>
+            <CustomTextInput 
+                style={{flex: 1}}
+                placeholder = 'Username'
```

Comment:
We can move the Username string to `constants` file

Reply:
done